cypress/page/address-step.page.ts

```diff
@@ -3,7 +3,7 @@ class AddressStepPage {
   private content: string;
 
   constructor() {
-      this.carNav = ".cart_navigation";
+      this.carNav = ".cart_navigation .button span";
```

Comment:
you don't need to change this selector, as you are using the get in combination with the contains

Reply:
Fixed